### DISCUSSION TITLE
Value Sequence automatic divider

### DISCUSSION THREAD

**Discussion by Llullcobain** [helgoboss/helgobox#844] [Ideas]
Hi, I'm starting to use Realearn, I feel it's very powerful, yet sometimes overwhelming, but I like challenges.

I'm mapping parameters of plugins, like for example Frequency potenciometer in the Waves API 550, and realize that I need to put the Value Sequence by hand, and need to figure it out which are the correct numbers for the correct positions.

Will be amazing if you could just put how many number of steps you have in the parameter (For example, you have 7 frequencies to choose, so you type 7, and automatically calculates the steps in the Value Sequence.

Thanks for all the work you do and Congrats!

**Comment by helgoboss** [comment]
You can do something similar: https://github.com/helgoboss/realearn/blob/master/doc/user-guide.adoc#target-value-sequence

**Comment by vonglan** [comment]
I simply move the target slider in the mapping, and find out the minimum interval between different values (e.g. 0.17). I then take this as step size.
